From 5cc5f7caa9da70b84bf665f613401eb74c398512 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:29 -0400
Subject: [redhat] kabi: Adding symbol param_ops_string (kernel/params.c)

Message-id: <70cdd751e1218fe6a7eadf68ba8aa40058d1adb8.1410510773.git.poros@redhat.com>
Patchwork-id: 96085
O-Subject: [PATCH RHEL7.1 149/233] kabi: Adding symbol param_ops_string (kernel/params.c)
Bugzilla: 1062361
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Not used by PowerPath directly

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_ops_string
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_ops_string
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_ops_string

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_ops_string b/redhat/kabi/kabi-module/kabi_ppc64/param_ops_string
new file mode 100644
index 0000000..009f9a4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_ops_string
@@ -0,0 +1,2 @@
+#1-
+0x4d405db8 param_ops_string vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_ops_string b/redhat/kabi/kabi-module/kabi_s390x/param_ops_string
new file mode 100644
index 0000000..009f9a4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_ops_string
@@ -0,0 +1,2 @@
+#1-
+0x4d405db8 param_ops_string vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_ops_string b/redhat/kabi/kabi-module/kabi_x86_64/param_ops_string
new file mode 100644
index 0000000..009f9a4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_ops_string
@@ -0,0 +1,2 @@
+#1-
+0x4d405db8 param_ops_string vmlinux EXPORT_SYMBOL
-- 
1.7.1